pontoon/base/models.py

```diff
@@ -2503,7 +2503,10 @@ class Entity(DirtyFieldsMixin, models.Model):
     resource = models.ForeignKey(Resource, models.CASCADE, related_name="entities")
     string = models.TextField()
     string_plural = models.TextField(blank=True)
-    key = models.TextField(blank=True)
+    # Unique identifier, used to compare DB and VCS objects
+    key = models.TextField()
```

Comment:
*question* This is probably going to be detected during testing on staging, but I wonder if there isn't an ancient bug/scenario which makes this field empty or null.

Reply:
I checked and there isn't.